src/components/organisms/offerContact/offerContact.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import { Map, TileLayer } from 'react-leaflet'
+import { Row, Col } from 'react-bootstrap'
+
+import './map.css'
+
+const OfferContact = ({offer}) => {
+
+  let latlng ={}
+
+  offer.location != null?
+  	latlng = {
+      lat: offer?.location?.latitude,
+      lng: offer?.location?.longitude
+    }
+  :
+  latlng = {
+	lat: 51,
+	lng: 30
+  };
+  return (
+	<Col>
+	  <Row className="p-4">
+	    <Col>
+		  <figure>
+		  	<img className="rounded-circle" src="https://gallery.dpcdn.pl/imgc/UGC/85487/g_-_-x-_-_-_x5b137a2c-9a30-4067-84fa-bc71678fab09.jpg" alt="not found" />
+		  </figure>
+		</Col>
+		<Col>
+		  <Row className="p-4">
+		    <h2>{offer?.seller?.name}</h2>
+		  </Row>
+		  <Row>
+			<Col className="p-0 m-0 text-center">
+		      <button className="btn btn-outline-secondary">
+				Call
+			  </button>
+			</Col>
+			<Col className="p-0 m-0 text-left">
+			  <button className="btn btn-outline-secondary">
```

Comment:
Może ładniej wyglądałoby z Buttonem z Bootstrapa (skoro używamy, to można korzystać :D)